src/main/kotlin/com/goodchoice/domain/image/model/ImageModificationRequest.kt

```diff
@@ -0,0 +1,5 @@
+package com.goodchoice.domain.image.model
+
+import java.net.URL
+
+data class ImageModificationRequest(val location: URL)
```

Comment:
`ImageDescriptor`
Image resource itself is not modifiable. Maybe `Uploading` instead of `Modificaton`?

Reply:
done